src/components/Blog/BlogListItem.tsx

```diff
@@ -26,10 +26,31 @@ export const BlogListItem: FunctionComponent<PostIndexItem> = ({
                     <a className={`d-block ${titleLinkClassName}`}>{frontmatter.title}</a>
                 </Link>
             </h1>
-            {frontmatter.author && frontmatter.publishDate && (
+            {frontmatter.authors?.length && (
                 <p className="text-align-center text-secondary mb-0">
-                    {frontmatter.author} on{' '}
-                    <time dateTime={frontmatter.publishDate}>{formatDate(frontmatter.publishDate)}</time>
+                    {frontmatter.authors.map((a, index) => (
+                        <span key={a.name}>
+                            {a.url ? (
+                                a.url.includes('http') ? (
+                                    <a href={a.url} target="_blank" rel="nofollow noreferrer">
+                                        {a.name}
+                                    </a>
+                                ) : (
+                                    <Link href={a.url}>{a.name}</Link>
+                                )
+                            ) : (
+                                a.name
+                            )}
+                            {/* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */}
+                            {index === frontmatter.authors!.length - 1 ? ' ' : ', '}
+                        </span>
+                    ))}
+
+                    {frontmatter.publishDate && (
+                        <p className="text-align-center text-secondary mb-0">
+                            <time dateTime={frontmatter.publishDate}>{formatDate(frontmatter.publishDate)}</time>
+                        </p>
```

Comment:
This is actually invalid HTML syntax. You can't nest p tags inside p tags.

Error in the console:
```
Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
```

I would suggest moving this down as a sibling of the other p tag and then wrapping the whole block in a fragment `<></>`

Reply:
Good catch! I had this for `PodcastListItem` & `PostcastPost` but forgot to replicate it here. Fixed in [62cdb04](https://github.com/sourcegraph/about/pull/5363/commits/62cdb04cc6397981bdf025958908acf30821064b)